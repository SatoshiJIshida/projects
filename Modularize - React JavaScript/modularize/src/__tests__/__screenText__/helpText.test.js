import React from "react";
import { shallow } from "enzyme";
import HelpText from "../../components/screenText/helpText";

/**
 * Library Jest taken from source https://jestjs.io/
 * Library Enzyme taken from source https://www.npmjs.com/package/enzyme
 */

describe("Help Text", () => {
  it("should be defined", () => {
    expect(HelpText).toBeDefined();
  });

  it("should render correctly", () => {
    const helpText = shallow(<HelpText name="helpText-test" />);
    expect(helpText).toMatchSnapshot();
  });
});
