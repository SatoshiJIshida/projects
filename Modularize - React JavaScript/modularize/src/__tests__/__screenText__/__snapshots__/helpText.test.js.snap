// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Help Text should render correctly 1`] = `ShallowWrapper {}`;
